src/Resources/assets/styles/_tailwind.scss

```diff
@@ -139,6 +139,11 @@
     @apply inline-flex items-center bg-primary-500 hover:bg-primary-700;
 }
 
+.whatwedo-crud-button--rounded {
+    @extend .whatwedo-utility-button;
+    @apply rounded-full p-1;
+}
+
```

Comment:
Nit: That really would be another case that goes into the Tailwind configuration. But at this point, everything seems to go away from the Tailwind principles so I guess, that is a higher order discussion to have later.